src/vos/evtree.c

```diff
@@ -1336,7 +1336,7 @@ evt_root_empty(struct evt_context *tcx)
 static int
 evt_root_tx_add(struct evt_context *tcx)
 {
-	void	*root;
+	struct evt_root	*root;
```

Comment:
Neither is a valgrind issue..they are real bugs caught by valgrind.  This one is more serious.